CHANGELOG.rst

```diff
@@ -1,5 +1,30 @@
 .. _changelog:
 
+0.34.2
+------
+
+Renku ``0.34.2`` fixes several bugs in the Gateway.
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+**🐞 Bug Fixes**
+
+-  **Gateway:** properly use Redis sentinel client
+   (`#668 <https://github.com/SwissDataScienceCenter/renku-gateway/issues/668>`__)
+   (`5ab4447 <https://github.com/SwissDataScienceCenter/renku-gateway/commit/5ab44475c9f7a516ddb8865c8f70db9bdb0ba5ec>`__)
+-  **Gateway:** properly redirect from /gitlab urls
+   (`#669 <https://github.com/SwissDataScienceCenter/renku-gateway/issues/669>`__)
+   (`2fac96f <https://github.com/SwissDataScienceCenter/renku-gateway/commit/2fac96f5c6141f4e57ae5cc77877670156bceae5>`__)
+-  **Gateway:** return 404 if the core service metadata does not exist instead of redirecting to latest
```

Comment:
Any chance you could reword slightly - I don't understand what it means...redirecting to latest what?

Reply:
Sure I will reword it. It is in case that the metadata version for the core service is not available or supported. Then we should return 404 rather than redirecting to the latest metadata version.